perception/autoware_object_sorter/src/object_sorter_base.cpp

```diff
@@ -97,27 +139,29 @@ void ObjectSorterBase<ObjsMsgType>::objectCallback(
     RCLCPP_WARN(this->get_logger(), "Transform lookup failed: %s", ex.what());
   }
 
+  double dist_origin_x = ego_pos.x + range_calc_offset_x_;
```

Comment:
[memo]
remember ego_pos means object_frame -> ego vector in object_frame coordinate.